lib/miq_apache/control.rb

```diff
@@ -60,7 +59,7 @@ def self.stop
     def self.run_apache_cmd(command)
       Dir.mkdir(File.dirname(APACHE_CONTROL_LOG)) unless File.exist?(File.dirname(APACHE_CONTROL_LOG))
       begin
-        res = MiqUtil.runcmd("apachectl", :params => [[command]])
+        res = AwesomeSpawn.run!("apachectl", :params => [command], :combined_output => true).output
```

Comment:
Clarification here on the `[[command]]`.  AwesomeSpawn doesn't require the double Array nesting, which is just a holdover from the old implementation of runcmd.

```ruby
[2] pry(main)> AwesomeSpawn.build_command_line("apachectl", ["start"])
=> "apachectl start"
[3] pry(main)> AwesomeSpawn.build_command_line("apachectl", [["start"]])
=> "apachectl start"
```